picard/ui/mainwindow.py

```diff
@@ -1480,6 +1490,32 @@ def check_for_update(self, show_always):
             callback=update_last_check_date
         )
 
+    def make_script_selector_menu(self):
+        config = get_config()
+        naming_scripts = config.setting["file_renaming_scripts"]
+        selected_script_id = config.setting["selected_file_naming_script_id"]
+        self.script_quick_selector_menu.clear()
+
+        def _add_menu_item(title, id):
+            script_action = QtWidgets.QAction(title, self.script_quick_selector_menu)
+            script_action.triggered.connect(partial(self.select_new_naming_script, id))
+            if id == selected_script_id:
+                script_action.setIcon(self.style().standardIcon(QtWidgets.QStyle.SP_DialogApplyButton))
```

Comment:
I wonder if it would be better to either do

```python
script_action.setCheckable(True)
script_action.setChecked(id == selected_script_id)
```

Or if we don't want to have every item with an empty checkbox then only:

```python
if id == selected_script_id:
    script_action.setCheckable(True)
    script_action.setChecked(True)
```

The icon works too, I guess. But maybe we need to check how this looks across platforms. I think it can even happen that menu icons get disabled by the platform, then it would not show up at all.
```

Reply:
I tried it with the check boxes on all lines and actually like that better than any of the other options I've explored.  Good suggestion.